src/main/java/org/jabref/logic/importer/fetcher/AstrophysicsDataSystem.java

```diff
@@ -154,8 +155,9 @@ public void doPostCleanup(BibEntry entry) {
         new FieldFormatterCleanup(FieldName.TITLE, new RemoveBracesFormatter()).cleanup(entry);
         new FieldFormatterCleanup(FieldName.AUTHOR, new NormalizeNamesFormatter()).cleanup(entry);
 
-        // Remove url to ADS page
+        // Remove ADS note
         new FieldFormatterCleanup("adsnote", new ClearFormatter()).cleanup(entry);
-        new FieldFormatterCleanup("adsurl", new ClearFormatter()).cleanup(entry);
+        // Move adsurl to url field
+        new MoveFieldCleanup("adsurl","url").cleanup(entry);
```

Comment:
Can you please use `FieldName.URL` instead of `"url"`. Moreover, add a space after the comma.